tf2/include/tf2/transform_datatypes.h

```diff
@@ -43,7 +44,6 @@ namespace tf2
 template <typename T>
 class Stamped : public T{
  public:
-  typedef std::chrono::system_clock::time_point TimePoint;
```

Comment:
This is new.  I determined that this was already being defined elsewhere (namely time.h), so we can just use that version instead.